docs/language.md

```diff
@@ -6001,6 +6195,21 @@ let counterRef = countCap.borrow()
 let counterRef2 = publicAccount.borrow<&Counter>(/storage/counter)
 ```
 
+### Account Creation
+
+Accounts can be created by calling the `AuthAccount` constructor
+and passing the account that should pay for the account creation for the `payer` parameter.
+
```

Comment:
```suggestion
The `payer` must have Flow tokens in their account to be able to create an account. The Flow tokens are automatically withdrawn from their account's token vault.
```

Reply:
Good point, added a less-specific variant of this